Lib/pprint.py

```diff
@@ -643,7 +643,7 @@ def _perfcheck(object=None):
         object = [("string", (1, 2), [3, 4], {5: 6, 7: 8})] * 100000
     p = PrettyPrinter()
     t1 = time.perf_counter()
-    p._safe_repr(object, {}, None, 0, True)
+    p._safe_repr(object, {}, None, 0)
```

Comment:
Thank for reporting. We typically commit a bugfix together with a unit test that will prevent the bug reoccurring. I'm not sure though whether _perfcheck is a feature that needs to be fixed (is it user facing?) or dead code that needs to be removed or a test that needs to move to the test module. 

Any idea what it's for?

Reply:
It isn't user facing and it isn't dead.  We sometime have code the is just for us in a main section.  For example, the *random* module also has some performance measurement code and example distributions — we use that sometimes while maintaining the module.

Reply:
I'd write a sanity test which verifies that `$ python -m pprint` works. More unit tests could then be added in the future if needed, but a simple sanity test would be enough.

From what I can tell, that is what didn't work for @ArturKhuziakhmetov when they opened #92546, and it makes sense to test. It is surely user-facing, and should work.

Reply:
Update: I just noticed I missed some of @rhettinger's comments. Since this is undocumented functionality, refraining from testing it may make sense. However, seeing broken code shipped in production builds raises a red flag for me.

Reply:
But the production code is really the functions that people import from the pprint module, not the unofficial script functionality.